.github/workflows/spack_build.yml

```diff
@@ -10,6 +10,7 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         python-version: [3.9]
+        compiler-version: [gcc@11.3.0]
```

Comment:
`gcc@12` doesn't build neither `py-torch@1.9` nor `py-torch@1.11`

Reply:
related to https://github.com/pytorch/pytorch/issues/77939 which refers to https://gcc.gnu.org/bugzilla/show_bug.cgi?id=105593

Reply:
Interesting and slightly strange.